.travis.yml

```diff
@@ -1,5 +1,25 @@
 language: haskell
```

Comment:
I'm not sure if it's important, but multi-ghc-travis [recommends not setting `language: haskell`](https://github.com/hvr/multi-ghc-travis/blob/c9c87d36c450d7f9cb3183dcaf1f77b60f916f28/.travis.yml#L1-L2).